### ISSUE TITLE
Sim focal plane footprints are weird now

### ISSUE THREAD

**Issue by mhasself** [simonsobs/sotodlib#508]
@keskitalo I believe #409 did change the default focal plane.  (cc @nad-dch , @kmharrington .)

The control of detailed orientation of subwafers is cool but this isn't how the tube is assembled.

SAT1 focal plane from commit 4d660fc (shows proper interlocking of wafers):

![image](https://github.com/simonsobs/sotodlib/assets/1054988/a1f774e7-a776-4805-8bd1-27f3eede10b3)

After https://github.com/simonsobs/sotodlib/pull/409 (commit cf6221f) (shows fun detailed orientations, but incorrect interlocking):

![image](https://github.com/simonsobs/sotodlib/assets/1054988/02494a99-9b18-40ec-adb9-7c8abec9262e)

The LAT is also affected:
![image](https://github.com/simonsobs/sotodlib/assets/1054988/58345fb1-e85d-4225-ab6a-7495e6fd083a)

**Comment by tskisner** [comment]
Hi @mhasself , I am right now in the middle of a working on a branch with more testing of the focalplane spacing and HWP angle / detector gamma checks.  I am comparing my branch against the layout generated by latest master HEAD (`dc14cd825`).  For SAT1 230GHz I get this layout:
[sat_f230.pdf](https://github.com/simonsobs/sotodlib/files/12130578/sat_f230.pdf)
Which seems to show correct positioning...

**Comment by tskisner** [comment]
I will investigate this a bit more in my branch- were your plots made by dumping SAT sims and then loading back in?  Happy to chat on slack if that is more efficient.

**Comment by mhasself** [comment]
No, they're based on SOFocalplane 'quat' entries.  I will check a sim.
```
import sotodlib.coords
from so3g.proj import quat
import sotodlib.toast as sotoast
import pylab as pl

def get_wafer_info(telescope, cache_file=None):
    focalplane = sotoast.SOFocalplane(telescope=telescope)
    qs = []
    for k in focalplane.keys():
        qs.append(focalplane[k]['quat'])
    return qs

tel = 'LAT'
x = get_wafer_info(tel)
qs = sotodlib.coords.ScalarLastQuat(x).to_g3()
xi, eta, gamma = quat.decompose_xieta(qs)

pl.scatter(xi, eta, s=2)
pl.savefig(f'fp_{tel}.png')
pl.clf()
```

**Comment by tskisner** [comment]
This should be fixed now in #509.  Running your script for both `LAT` and `SAT1` gives the attached plots:
![fp_SAT1_so3g](https://github.com/simonsobs/sotodlib/assets/84221/ddb6ac9f-b6ae-4dfb-bd6c-42ef06933964)
![fp_LAT_so3g](https://github.com/simonsobs/sotodlib/assets/84221/4238768c-a9d2-45ad-a0b2-6d0dfd860ed5)
